GMPWrapper/scripts/convertMarlinSteeringToGaudi.py

```diff
@@ -55,6 +56,64 @@ def getProcessors(tree):
   return processors
 
 
+# Replace with constants if pattern found in value(s)
+def replaceConstants(value, constants):
+  formatted_array = []
+  split_values = value.split()
+  for val in split_values:
+    captured_patterns = re.findall('\$\{\w*\}', val)
+    if len(captured_patterns) == 0:
+      formatted_array.append("\"{}\"".format(val))
+    elif len(captured_patterns) >= 1:
+      val_format = re.sub(r'\$\{(\w*)\}', r'%(\1)s', val)
+      val_format = "\"{}\" % CONSTANTS".format(val_format)
+      formatted_array.append(val_format)
+
+  return ", ".join(formatted_array)
+
+
+# Find constant tags, write them to python and replace constants within themselves
+def convertConstants(lines, tree):
+  constants = dict()
+
+  constElements = tree.findall('constants/include')
+  if constElements:
+    print('WARNING: include statements found inside constants tag')
+
+  constElements = tree.findall('constants/constant')
+  for const in constElements:
+    constants[const.attrib.get('name')] = getValue(const)
+
+  for key,value in constants.items():
+    if value:
```

Comment:
```suggestion
    if not value:
      continue
```
And dedent the rest of the loop block

Reply:
Isn't it the same though? it has to branch anyway, and only do something if there is a value.

Reply:
Yes it is the same, but easier to parse visually with shorter blocks and not these long blocks inside if and no else.